### DISCUSSION TITLE
With Android 13 you cannot enable the accessibility service

### DISCUSSION THREAD

**Discussion by FBoffiS1** [rustdesk/rustdesk#6225] [General]
### Bug Description

Good morning, everyone,

I am also trying rustdesk on some zebra technologies terminals with android 13 but in none of them can I enable the rustdesk service in accessibility as you can see from the attached screens.

Is it possible to bypass this block?

![Screenshot_20231030_092902](https://github.com/rustdesk/rustdesk/assets/72557406/ebb75736-500f-4b20-8968-e3008e39e333)
![Screenshot_20231030_092852](https://github.com/rustdesk/rustdesk/assets/72557406/5507e47b-96ee-4093-9d2f-800971a5f418)

Thanks,

### How to Reproduce

Install rustdesk apk, turn on input control flag and try to start accessibility rustdesk service

### Expected Behavior

having the possibility of enabling the service as it was on the old operating systems

### Operating system(s) on local side and remote side

Android 13

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

![Screenshot_20231030_092902](https://github.com/rustdesk/rustdesk/assets/72557406/f6cbb669-f6db-4af1-9bf8-f6f8b7dc2a54)
![Screenshot_20231030_092852](https://github.com/rustdesk/rustdesk/assets/72557406/f85a24a6-c322-4d09-8a46-d5ff80106a2f)

**Comment by xlionjuan** [comment]
Yes
![Screenshot_20231031-133240.png](https://github.com/rustdesk/rustdesk/assets/25243894/4f3f8b22-2e32-4faf-93dd-54fa2c6a39bc)

**Reply by FBoffiS1** [reply]
Thans... it worked
